sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -20,12 +20,9 @@ package org.apache.spark.sql
 import java.io.{File, FilenameFilter}
 import java.nio.file.{Files, Paths}
 import java.time.{Duration, LocalDateTime, Period}
-
```

Comment:
let's revert the unnecessary changes in this file

Reply:
done